### PR TITLE
Updated command executed message

### PR DESCRIPTION
**Pull request by CoderQCFromScraatch**
tl;dr I changed it from an "Upload Failed" to a "Command Successfully Executed!"

### CODE REVIEW

File: incl/comments/uploadGJComment.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 $uploadDate = time();
 $decodecomment = base64_decode($comment);
 if(Commands::doCommands($id, $decodecomment, $levelID)){
-	exit("-1");
+	exit("temp_0_Executed command successfully!");
```
Comment (EMREOYUN) [inline incl/comments/uploadGJComment.php:23]: This is not how the PHP exit() function works in Geometry Dash. It must be an integer, not a string.

**Review by EMREOYUN — CHANGES_REQUESTED** [review]
~~This is not how the PHP exit() function works in Geometry Dash. It must be an integer, not a string.~~

Issue resolved.

File: incl/comments/uploadGJComment.php (hunk 1 above)
Comment (Cvolton) [reply incl/comments/uploadGJComment.php:23]: that is not true, comment bans use this format

Comment (EMREOYUN) [reply incl/comments/uploadGJComment.php:23]: Really? Then is the `temp_0` part necessary?

Comment (Cvolton) [reply incl/comments/uploadGJComment.php:23]: It is, `temp_0` tells GD that it's a temporary comment ban for 0 seconds and "Executed command succesfully!" will then show up as the ban reason.

Comment (EMREOYUN) [reply incl/comments/uploadGJComment.php:23]: Well the game is changed the last time I left at. So I see no issues.

**Review by EMREOYUN — APPROVED** [review]

File: incl/comments/uploadGJComment.php (hunk 1 above)
Comment (CoderQCFromScraatch) [reply incl/comments/uploadGJComment.php:23]: The PHP exit function just returns a value of any type and quits.

**Comment by CoderQCFromScraatch** [comment]
I did have that problem , but i fixed it with some code modification.

Including all of my dumb commits might fix it.

**Comment by MegaSa1nt** [comment]
i tested, it works fine!

**Comment by Cvolton** [comment]
the code in the PR doesn't work exactly for the reasons I suspected but I implemented the idea in b9fd8e722e1edfcf075ce3f3a36637cd5209677e now

**Pull request closed without merging**